Moving to vitepress (#138)
Paperback-iOS/website: src/tools/index.md

# Tools

Welcome to the tools section. Below, you'll find a collection of tools designed to enhance your experience with the Paperback app. These tools are conveniently accessible through this website and feature user-friendly interfaces.

## Backup Converter - _Coming soon!_

> Have you switched between apps or plan to do so? We recognize the significance of preserving your progress. Our Backup Converter tool facilitates this process. With this tool, effortlessly convert backups from other popular comic/manga reader apps into a compatible format for Paperback, and vice versa.
> 
> Supported readers:
> 
> - None
> 
> [Go to Backup Converter](/tools/backup-converter)

Author: ```suggestion
Supported readers:
- None

[Go to Backup Converter](/tools/backup-converter)
```

> ## Themes Creator - _Coming soon!_
> 
> ::: tip Already Available Themes
> Not interested in creating themes yourself but still want to change the appearance of your app? Simply browse our list of publicly available themes [here](/guides/themes) and install your favorite one.
> :::
> 
> Customize your Paperback experience with our Themes Creator tool. This intuitive utility allows you to explore, create, and modify themes for your app effortlessly. Whether you prefer a subdued dark theme for nighttime reading or a lively theme to match your mood, the Themes Creator offers a wide range of possibilities.
> 
> :::info Current Alternatives
> There are alternative non-web-app tools available while our Themes Creator tool is under development:
> 
> 1. [Paperback Themes:](https://github.com/Celarye/Paperback-themes) Offers an advanced command-line interface program for creating themes.
> 2. [Paperback Themes GUI:](https://github.com/LucifersCircle/paperback-themes-gui) A user-friendly graphical interface for creating themes.
> 
> :::
> 
> [Go to Themes Creator](/tools/themes-creator)

Author: ```suggestion
## Theme Creators

::: tip Already Available Themes
Not interested in creating themes yourself but still want to change the appearance of your app? Simply browse our list of publicly available themes [here](/guides/themes) and install your favorite one.
:::

### Community Theme Generators
There are alternative non-web-app tools available while our Themes Creator tool is under development:

1. [Paperback Themes:](https://github.com/Celarye/Paperback-themes) Offers an advanced command-line interface program for creating themes.
2. [Paperback Themes GUI:](https://github.com/LucifersCircle/paperback-themes-gui) A user-friendly graphical interface for creating themes.

[Go to Themes Creator](/tools/themes-creator)
```
